classes/submission/Collector.php

```diff
@@ -250,6 +251,17 @@ public function filterByRevisionsSubmitted(?bool $revisionsSubmitted): AppCollec
         return $this;
     }
 
+    /**
+     * Limit results to only submissions with the specified IDs
+     *
+     * @param int[] $submissionIds Submission IDs
+     */
+    public function filterBySubmissionIds(array $submissionIds): static
```

Comment:
I think `null` should be allowed to disable the filter.

```suggestion
     * @param ?int[] $submissionIds Submission IDs
     */
    public function filterBySubmissionIds(?array $submissionIds): static
```

Reply:
updated